docs/conf/australia/2021/news/cfp-open.rst

```diff
@@ -0,0 +1,48 @@
+:template: {{year}}/generic.html
+
+.. post:: June 20, 2021
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
```

Comment:
iirc this needs to be literal `australia-2021`

Reply:
I thought so too, but given that the post does appear on https://writethedocs-www--1546.org.readthedocs.build/conf/australia/2021/news/ it might be ok this way?